sdk/python/feast/client.py

```diff
@@ -493,17 +497,70 @@ def get_feature_set(
             raise grpc.RpcError(e.details())
         return FeatureSet.from_proto(get_feature_set_response.feature_set)
 
-    def list_entities(self) -> Dict[str, Entity]:
+    def list_features(
+        self,
+        project: str = None,
+        entities: List[str] = list(),
+        labels: Dict[str, str] = dict(),
+        return_as_dict=True,
```

Comment:
I think it will be cleaner to use a different method instead of this argument. `list_features_by_ref`, even though it's not as clean.

Reply:
Would you want to do away with list_features all together for now?